python/paddle/v2/event.py

```diff
@@ -3,24 +3,54 @@
 
 There are:
 
-* BeginTraining
```

Comment:
Begin & End Training is not useful because users can write their own code before/after invoking `train` method.